pkgs/top-level/all-packages.nix

```diff
@@ -10976,23 +10976,13 @@ with pkgs;
 
   witness = callPackage ../tools/security/witness { };
 
-  openconnect = openconnect_gnutls;
+  openconnectPackages = dontRecurseIntoAttrs (callPackage ../tools/networking/openconnect { });
 
-  openconnect_openssl = callPackage ../tools/networking/openconnect {
-    inherit (darwin.apple_sdk.frameworks) PCSC;
-    gnutls = null;
-  };
+  inherit (openconnectPackages) openconnect openconnect_unstable openconnect_openssl;
 
-  openconnect_gnutls = callPackage ../tools/networking/openconnect {
-    inherit (darwin.apple_sdk.frameworks) PCSC;
-    openssl = null;
-  };
+  openconnect_head = openconnect_unstable;
```

Comment:
This should be moved to aliases if we need it at all.

Reply:
You're right, they are probably not needed anymore. I've converted them to `throw`s.